flask/app/templates/base.html

```diff
@@ -5,6 +5,7 @@
     <meta charset="utf-8">
     <meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
     <link rel="stylesheet" href="./static/styles/index.css" />
+    <link rel="stylesheet" href="./static/styles/play.css" />
```

Comment:
Please remove this line

Any css defined in `base.html` will be loaded by every page that is based on it. `play.css` should only be loaded for the Play page, not for every page.

Reply:
done 👍🏼